contracts/libraries/SafeMigration.sol

```diff
@@ -28,7 +28,7 @@ contract SafeMigration is SafeStorage {
      */
     address public immutable SAFE_L2_SINGLETON;
     /**
-     * @notice Addresss of the Fallback Handler
+     * @notice Address of the Fallback Handler
      */
     address public immutable SAFE_FALLBACK_HANDLER;
```

Comment:
For this file, we backported it to import the full `Safe` contract just for calling the `setFallbackHandler` method on itself. When verifying the contracts in a block explorer, this will pull in a lot of additional code which isn't great IMO. Additionally, it makes inheriting the contracts harder in other projects.

Can we use the same strategy as we did in `SafeToL2Migration` and add an `ISafe` interface local to this file with just the methods that we need to call?

Reply:
Done 👍🏾